pkg/platform/provider/baremetal/cluster/kubeadm.go

```diff
@@ -126,6 +126,10 @@ func (p *Provider) getInitConfiguration(c *v1.Cluster) *kubeadmv1beta2.InitConfi
 			nodeRegistration.Name = machineIP
 		}
 	}
+
+	// Specify infra(pause) pod version
+	kubeletExtraArgs["pod-infra-container-image"] = p.config.Registry.Prefix + "/pause:3.2"
```

Comment:
we have method to get images full name, don't hardcode image name

Reply:
if upgrade from 1.20.4 to 1.21.4, the pause will be which version?

Reply:
pause version is 3.2 in k8s 1.20.x/1.19.x by default with all container runtimes,

and pause version in 3.2 controllered by config in code after k8s 1.21 with all container runtimes,

In a word, in k8s 1.19-1.21  pause version is `3.2` @leoryu